indexer-cli didn't work v0.20.11

The following error occurred when starting indexer-cli version 0.20.11 and running graph indexer status.
The axios module does not seem to be included.

```
/opt/indexer/node_modules/gluegun/build/index.js:13
    throw up;
    ^

Error: Cannot find module 'axios'
Require stack:
- /opt/indexer/packages/indexer-common/dist/allocations/query-fees.js
- /opt/indexer/packages/indexer-common/dist/allocations/index.js
- /opt/indexer/packages/indexer-common/dist/actions.js
- /opt/indexer/packages/indexer-common/dist/index.js
- /opt/indexer/packages/indexer-cli/dist/actions.js
- /opt/indexer/packages/indexer-cli/dist/commands/indexer/actions/approve.js
- /opt/indexer/node_modules/gluegun/build/loaders/module-loader.js
- /opt/indexer/node_modules/gluegun/build/loaders/command-loader.js
- /opt/indexer/node_modules/gluegun/build/runtime/runtime.js
- /opt/indexer/node_modules/gluegun/build/domain/builder.js
- /opt/indexer/node_modules/gluegun/build/index.js
- /opt/indexer/packages/indexer-cli/dist/cli.js
- /opt/indexer/packages/indexer-cli/bin/graph-indexer
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/opt/indexer/packages/indexer-common/dist/allocations/query-fees.js:16:33)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/opt/indexer/packages/indexer-common/dist/allocations/query-fees.js',
    '/opt/indexer/packages/indexer-common/dist/allocations/index.js',
    '/opt/indexer/packages/indexer-common/dist/actions.js',
    '/opt/indexer/packages/indexer-common/dist/index.js',
    '/opt/indexer/packages/indexer-cli/dist/actions.js',
    '/opt/indexer/packages/indexer-cli/dist/commands/indexer/actions/approve.js',
    '/opt/indexer/node_modules/gluegun/build/loaders/module-loader.js',
    '/opt/indexer/node_modules/gluegun/build/loaders/command-loader.js',
    '/opt/indexer/node_modules/gluegun/build/runtime/runtime.js',
    '/opt/indexer/node_modules/gluegun/build/domain/builder.js',
    '/opt/indexer/node_modules/gluegun/build/index.js',
    '/opt/indexer/packages/indexer-cli/dist/cli.js',
    '/opt/indexer/packages/indexer-cli/bin/graph-indexer'
  ]
}
```

I could reproduce this by building the `indexer-cli` image on current `main` and executing the provided binary:

```
graph@78713798bfac:/$ ./opt/indexer/packages/indexer-cli/bin/graph-indexer                                                                                                     
/opt/indexer/node_modules/gluegun/build/index.js:13
    throw up;
    ^

Error: Cannot find module 'axios'
Require stack:
- /opt/indexer/packages/indexer-common/dist/allocations/query-fees.js
- /opt/indexer/packages/indexer-common/dist/allocations/index.js
- /opt/indexer/packages/indexer-common/dist/actions.js
- /opt/indexer/packages/indexer-common/dist/index.js
- /opt/indexer/packages/indexer-cli/dist/actions.js
- /opt/indexer/packages/indexer-cli/dist/commands/indexer/actions/approve.js
- /opt/indexer/node_modules/gluegun/build/loaders/module-loader.js
- /opt/indexer/node_modules/gluegun/build/loaders/command-loader.js
- /opt/indexer/node_modules/gluegun/build/runtime/runtime.js
- /opt/indexer/node_modules/gluegun/build/domain/builder.js
- /opt/indexer/node_modules/gluegun/build/index.js
- /opt/indexer/packages/indexer-cli/dist/cli.js
- /opt/indexer/packages/indexer-cli/bin/graph-indexer
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/opt/indexer/packages/indexer-common/dist/allocations/query-fees.js:16:33)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/opt/indexer/packages/indexer-common/dist/allocations/query-fees.js',
    '/opt/indexer/packages/indexer-common/dist/allocations/index.js',
    '/opt/indexer/packages/indexer-common/dist/actions.js',
    '/opt/indexer/packages/indexer-common/dist/index.js',
    '/opt/indexer/packages/indexer-cli/dist/actions.js',
    '/opt/indexer/packages/indexer-cli/dist/commands/indexer/actions/approve.js',
    '/opt/indexer/node_modules/gluegun/build/loaders/module-loader.js',
    '/opt/indexer/node_modules/gluegun/build/loaders/command-loader.js',
    '/opt/indexer/node_modules/gluegun/build/runtime/runtime.js',
    '/opt/indexer/node_modules/gluegun/build/domain/builder.js',
    '/opt/indexer/node_modules/gluegun/build/index.js',
    '/opt/indexer/packages/indexer-cli/dist/cli.js',
    '/opt/indexer/packages/indexer-cli/bin/graph-indexer'
  ]
}
```

--- a/packages/indexer-common/package.json
+++ b/packages/indexer-common/package.json
@@ -27,6 +27,7 @@
     "@thi.ng/heaps": "1.2.38",
     "@urql/core": "2.4.4",
     "@urql/exchange-execute": "1.2.2",
+    "axios": "0.26.1",
     "body-parser": "1.19.1",
     "cors": "2.8.5",
     "ethers": "5.7.0",

